docs/src/release-notes-js.md

```diff
@@ -6,6 +6,46 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.50
+
+### Test runner
```

Comment:
I'd include two snippets here:
- `test.step.skip()` with a timeout and attachment;
- `npx playwright test --update-snapshots=changed --update-source-method=3way`
And group the items accordingly.